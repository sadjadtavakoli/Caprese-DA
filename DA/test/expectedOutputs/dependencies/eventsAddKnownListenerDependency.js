{"adderFunction-test/unit_tests/eventsaddknownlistenerdependency.js-8-10":{"callers":[],"tests":["t_1","t_61"]},"adderFunction2-test/unit_tests/eventsaddknownlistenerdependency.js-13-15":{"callers":[],"tests":["t_1"]},"thidOne-test/unit_tests/eventsaddknownlistenerdependency.js-20-22":{"callers":[],"tests":["t_1"]},"listner1-test/unit_tests/eventsaddknownlistenerdependency.js-4-6":{"callers":[],"tests":["t_61"]},"keyMap":{"t_1":"eventsaddknownlistenerdependency.js-test/unit_tests/eventsaddknownlistenerdependency.js-1-24","t_41":"adderFunction-test/unit_tests/eventsaddknownlistenerdependency.js-8-10","t_56":"listner1-test/unit_tests/eventsaddknownlistenerdependency.js-4-6","t_57":"adderFunction2-test/unit_tests/eventsaddknownlistenerdependency.js-13-15","t_61":"thidOne-test/unit_tests/eventsaddknownlistenerdependency.js-20-22","t_22":"utils.js-utils.js-15-18"}}