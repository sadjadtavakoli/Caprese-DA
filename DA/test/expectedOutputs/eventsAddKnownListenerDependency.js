
#8 function adderFunction-test/unit_tests/eventsaddknownlistenerdependency.js-8-10 entered from eventsAddKnownListenerDependency.js-test/unit_tests/eventsaddknownlistenerdependency.js-1-13
#end Execution