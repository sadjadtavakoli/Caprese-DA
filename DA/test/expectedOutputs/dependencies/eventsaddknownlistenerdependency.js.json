{"adderFunction-test/unit_tests/eventsaddknownlistenerdependency.js-8-10":{"impacted":["t_12"]},"thidOne-test/unit_tests/eventsaddknownlistenerdependency.js-17-19":{"impacted":["t_4","t_7"]},"keyMap":{"t_1":"eventsaddknownlistenerdependency.js-test/unit_tests/eventsaddknownlistenerdependency.js-1-23","t_4":"adderFunction-test/unit_tests/eventsaddknownlistenerdependency.js-8-10","t_7":"listner1-test/unit_tests/eventsaddknownlistenerdependency.js-4-6","t_8":"adderFunction2-test/unit_tests/eventsaddknownlistenerdependency.js-13-15","t_12":"thidOne-test/unit_tests/eventsaddknownlistenerdependency.js-17-19"}}